### ISSUE TITLE
I really love your mod but I've encountered an endgame issue:

### ISSUE THREAD

**Issue by hardavan** [AlexModGuy/Ice-and-Fire-Suggestions#1390]
HOW THE DUCK I AM SUPPOSED TO FIND THOSE [hot ladies] DRAGONS IN THEIR LAIRS? 
I am serious! I can't find any dragon 4 or 5 stage by myself please make cartographer or chronicler sell a map of the nearest nest with dragon caverns?

**Comment by Saber130** [comment]
Yes they should make villagers sell dragon cave locations, or just install a mod that locates dragons
